packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1195,6 +1207,7 @@ abstract class ColumnRenderer<T> {
     BuildContext context,
     T data, {
     bool isRowSelected = false,
+    bool isRowHovered = false,
```

Comment:
add a note to this dart doc that states that `isRowHovered` is only used when `enableHoverHandling` is true on the table that this column belongs to.